src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -256,12 +256,6 @@ void KaleidoScope_SetupPlayerPreRender(PlayState* play) {
     CLOSE_DISPS(play->state.gfxCtx, "../z_kaleido_scope_PAL.c", 509);
 }
 
-void KaleidoScope_ProcessPlayerPreRender(void) {
```

Comment:
Removing this here isn't entirely correct since it also calls the destroy function.